internal/transform/schema.go

```diff
@@ -72,51 +76,68 @@ type OperationOutput struct {
 
 // Details is a struct that provides additional information about operations in a way that aligns with the details struct in the BigQuery table history_operations
 type Details struct {
-	Account            string   `json:"account"`
-	Amount             float64  `json:"amount"`
-	AssetCode          string   `json:"asset_code"`
-	AssetIssuer        string   `json:"asset_issuer"`
-	AssetType          string   `json:"asset_type"`
-	Authorize          bool     `json:"authorize"`
-	BuyingAssetCode    string   `json:"buying_asset_code"`
-	BuyingAssetIssuer  string   `json:"buying_asset_issuer"`
-	BuyingAssetType    string   `json:"buying_asset_type"`
-	From               string   `json:"from"`
-	Funder             string   `json:"funder"`
-	HighThreshold      uint32   `json:"high_threshold"`
-	HomeDomain         string   `json:"home_domain"`
-	InflationDest      string   `json:"inflation_dest"`
-	Into               string   `json:"into"`
-	Limit              float64  `json:"limit"`
-	LowThreshold       uint32   `json:"low_threshold"`
-	MasterKeyWeight    uint32   `json:"master_key_weight"`
-	MedThreshold       uint32   `json:"med_threshold"`
-	Name               string   `json:"name"`
-	OfferID            int64    `json:"offer_id"`
-	Path               []Path   `json:"path"`
-	Price              float64  `json:"price"`
-	PriceR             Price    `json:"price_r"`
-	SellingAssetCode   string   `json:"selling_asset_code"`
-	SellingAssetIssuer string   `json:"selling_asset_issuer"`
-	SellingAssetType   string   `json:"selling_asset_type"`
-	SetFlags           []int32  `json:"set_flags"`
-	SetFlagsString     []string `json:"set_flags_s"`
-	SignerKey          string   `json:"signer_key"`
-	SignerWeight       uint32   `json:"signer_weight"`
-	SourceAmount       float64  `json:"source_amount"`
-	SourceAssetCode    string   `json:"source_asset_code"`
-	SourceAssetIssuer  string   `json:"source_asset_issuer"`
-	SourceAssetType    string   `json:"source_asset_type"`
-	SourceMax          float64  `json:"source_max"`
-	StartingBalance    float64  `json:"starting_balance"`
-	To                 string   `json:"to"`
-	Trustee            string   `json:"trustee"`
-	Trustor            string   `json:"trustor"`
-	Value              string   `json:"value"` // base64 encoding of bytes for operations that manage data
-	ClearFlags         []int32  `json:"clear_flags"`
-	ClearFlagsString   []string `json:"clear_flags_s"`
-	DestinationMin     string   `json:"destination_min"`
-	BumpTo             string   `json:"bump_to"`
+	Account            string     `json:"account"`
```

Comment:
❓   what're your thoughts on refactoring the way we're doing this `Details` field. I'm not sure why we originally decided to do it this way, but to me it seems pretty unwieldy (both as Go code, and in big query). Could we break it up (would be in a different PR of course)? Or is there a reason the fields need to be grouped like this? (eg. other projects are dependent on the fields being at the same level in `.details` or something like that)

 By looking at it these are the added fields correct?
`AccountID`, `BalanceID`, `ClaimableBalanceID`, `Claimants`, `DataAccountID`, `DataName`, `SignerAccountID`, `SignerKey`, `TrustlineAccountID`, `TrustlineAsset`

nit: for PR readability if you could point out the added fields it would be helpful, took me a minute to compare to see which ones were added 🙏 😄

Reply:
I dislike the current `Details` struct and had written a [new issue](https://github.com/stellar/hubble/issues/179) to restructure it when I fix the null handling. The way we currently structure `Details` does not allow for delineation between empty values and zero values, so the data does not match what is written to Horizon DB.

My hope was to refactor this to follow the Operations Protocol Design [here](https://github.com/stellar/go/blob/master/protocols/horizon/operations/main.go)... @acharb thoughts on us using this structure instead of our current one? Or were you thinking of flattening the `Details` object into a denormalized table in BQ?

Reply:
Other fields added: `BeginSponsor`, and `SponsoredID`

Reply:
resolving this since tracked in this ticket https://github.com/stellar/hubble/issues/179